### PR TITLE
slow tests: adding cloud init network data to alpine VM

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:
As part of the passt effort all the istio tests were moved to use Alpine.
The alpine VM was created without cloud-init network data which caused the VM startup to be very slow (it added ~2 hours per
network lane).
Adding network data to the alpine VMs to workaround the issue.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by AlonaKaplan** [comment]
@qinqon do you have an idea why the alpine VMs startup is so slow without cloud-init network data?

**Review by EdDev — COMMENTED** [review]
Thank you! Lets save the CI.

It will be nice to explain that the empty cloud init network config defaults to DHCPv4 and DHCPv6 enabled clients.
(BTW, we do not run this on the single IPv4/IPv6 lane? will it work there too?)

nit: The commit message subject like is a bit too long.

**Review by EdDev — APPROVED** [review]
Thank you!

**Review by EdDev — APPROVED** [review]

**Review by phoracek — APPROVED** [review]
/approve

File: tests/network/vmi_passt.go [hunk 1]
```diff
@@ -50,9 +50,11 @@ var _ = SIGDescribe("[Serial] Passt", func() {
 
 		networkData, err = libnet.NewNetworkData(
 			libnet.WithEthernet("eth0",
+				libnet.WithDHCP4Enabled(),
 				libnet.WithDHCP6Enabled(),
 			),
 		)
+		util.PanicOnError(err)
```
Comment (oshoval) [inline tests/network/vmi_passt.go:57]: why not Expect ?

**Pull request merged**